app/src/main/java/fr/free/nrw/commons/OkHttpConnectionFactory.java

```diff
@@ -41,7 +41,7 @@ private static OkHttpClient createClient() {
 
     private static HttpLoggingInterceptor getLoggingInterceptor() {
         final HttpLoggingInterceptor httpLoggingInterceptor = new HttpLoggingInterceptor()
-            .setLevel(Level.BASIC);
+            .setLevel(Level.BODY);
```

Comment:
I believe this should be reverted before this PR gets merged.

Reply:
Updated.